core/lib/multivm/src/versions/vm_latest/tests/evm_simulator.rs

```diff
@@ -1483,6 +1483,176 @@ fn test_basic_jumpi_vectors() {
     );
 }
 
+#[test]
+fn test_basic_address_vectors() {
+    assert_eq!(
+        test_evm_vector(
+            vec![
+                // push1 0xFF
+                hex::decode("60").unwrap(),
+                hex::decode("FF").unwrap(),
```

Comment:
Is this push needed in the test?

Reply:
I put them so if address opcode don't push anything to stack, it don't do weird things